### PR TITLE
churches-tierney-ozcoviz-dashboard-covidrecon-package.yml

### PR DESCRIPTION
**Pull request by timchurches**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
-  ~~Make sure "Allow edits from maintainers" is checked~~
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/churches-tierney-ozcoviz-dashboard-covidrecon-package.html
- [ ] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

File: contest/contributions/churches-tierney-ozcoviz-dashboard-covidrecon-package.yml
Comment (riccardoporreca) [inline contest/contributions/churches-tierney-ozcoviz-dashboard-covidrecon-package.yml]:
```suggestion
title: "OzCoViz Dashboard and the covidrecon Package"
```

File: contest/contributions/churches-tierney-ozcoviz-dashboard-covidrecon-package.yml
Comment (riccardoporreca) [inline contest/contributions/churches-tierney-ozcoviz-dashboard-covidrecon-package.yml:17]:
```suggestion
content: https://cbdrh.github.io/ozcoviz/
date: 2020-05-22-22:00
```

**Review by riccardoporreca — COMMENTED** [review]
@timchurches, edit from maintainers are not allowed, can you please make sure they are or commit the suggested changes?

**Comment by timchurches** [comment]
Requested changes approved. "Allow edits from maintainers" does not appear as an option when submitting a PR from a GitHub corporate account (it appears when submitting a PR from a personal account).

**Comment by riccardoporreca** [comment]
Thanks @timchurches, Note there are currently issues with GitHub preventing the integration of contributions (https://www.githubstatus.com/).
We will finalize submissions once such issues are solved.

**Review by riccardoporreca — APPROVED** [review]

**Comment by riccardoporreca** [comment]
Integration of contributions in the CovidR Gallery still blocked by an ongoing GitHub incident

**Comment by riccardoporreca** [comment]
Integration of contributions in the CovidR Gallery still blocked by an ongoing GitHub incident

**Comment by riccardoporreca** [comment]
@timchurches, the major GitHub incident was solved and now contributions are fully integrated.

Your contribution is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/churches-tierney-ozcoviz-dashboard-covidrecon-package.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Pull request merged**
